src/index.js

```diff
@@ -0,0 +1,110 @@
+import places from 'places.js';
+import GetWeatherData from './components/WeatherData';
+import ProcessData from './components/ProcessData';
+import GetIconImage from './components/GetIconImage';
+import Header from './components/Header';
+import Main from './components/Main';
+import Footer from './components/Footer';
+import './css/style.css';
+
+const page = document.querySelector('#content');
+page.append(Header(), Main(), Footer());
+
+const updateView = async ({
+  temp,
+  name,
+  country,
+  icon,
+  weatherDesc,
+  humidity,
+  tempMin,
+  tempMax,
+  wind,
+}) => {
+  const tempData = document.querySelector('#temp-data');
+  const city = document.querySelector('#city-span');
+  const min = document.querySelector('#min span');
+  const max = document.querySelector('#max span');
+  const humiditySpan = document.querySelector('#humidity span');
+  const windSpeed = document.querySelector('#wind span');
+  const weather = document.querySelector('#weather span');
+
+  tempData.innerText = `${Math.round(temp)}°`;
+  city.innerText = `${name}, ${country}`;
+  weather.innerText = weatherDesc;
+  min.innerText = `${tempMin} °C`;
+  max.innerText = `${tempMax} °C`;
+  humiditySpan.innerText = `${humidity} %`;
+  windSpeed.innerText = `${wind} m/s`;
```

Comment:
**(Note: we need to toggle not display both or one at the same time)**
Your app should have a toggle control to determine if the result will be shown in Celsius or Fahrenheit (According to the Odin instructions, the screenshot below;)
![image](https://user-images.githubusercontent.com/31897434/69664063-e2356180-1087-11ea-8e7d-e94e88863eb0.png)

https://www.theodinproject.com/courses/javascript/lessons/weather-app